tests/end2end/features/misc.feature

```diff
@@ -537,3 +537,30 @@ Feature: Various utility commands.
         And I put "foo" into the clipboard
         And I run :message-info {clipboard}bar{url}
         Then the message "foobarhttp://localhost:*/hello.txt" should be shown
+
+    # :debug-log-filter
+
+    Scenario: Using :debug-log-filter with url argument
+        When I run :debug-log-filter url
+        And I run :open url
+        And I wait until command is loaded
+        Then the page should contain log records with url messages
```

Comment:
All those "commands" don't really exist yet. I think the best way to do this is to set up a log filter with a category like `js`, do `:message-info this should not be logged` and then check that isn't on the page.